package.json

```diff
@@ -14,6 +14,7 @@
     "eslint": "8.39.0",
     "eslint-config-next": "13.3.1",
     "file-saver": "^2.0.5",
+    "jotai": "^2.1.0",
```

Comment:
Do you use jotai? 🤔  If not I would recommend to remove the package.

Reply:
i tried something what didnt work but iam going to implement it later anyway so it just chills here for now...